app/src/main/java/com/example/dinhh/soundscape/data/Model.kt

```diff
@@ -0,0 +1,16 @@
+package com.example.dinhh.soundscape.data
+
+import com.example.dinhh.soundscape.presentation.ListItem
+
+object Model {
+    val category: kotlin.collections.MutableList<ListItem> = java.util.ArrayList()
```

Comment:
Should import MutableList or ArrayList (basically if you use Kotlin, use MutableList) instead of `kotlin.collections.MutableList<ListItem>`